microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -148,18 +149,21 @@ def dependencies(self) -> List[Type[Tool]]:
         return [Git, Wget, Lscpu]
 
     def get_dpdk_version(self) -> VersionInfo:
+        self.node.log.debug(f"Found DPDK version {str(self._dpdk_version_info)}.")
         return self._dpdk_version_info
 
+    def has_dpdk_version(self) -> bool:
+        return bool(self._dpdk_version_info > "0.0.0")
+
     def has_tx_ip_flag(self) -> bool:
-        dpdk_version = self.get_dpdk_version()
-        if not dpdk_version:
+        if not self.has_dpdk_version():
             fail(
                 "Test suite bug: dpdk version was not set prior "
                 "to querying the version information."
             )
 
         # black doesn't like to direct return VersionInfo comparison
-        return bool(dpdk_version >= "19.11.0")  # appease the type checker
+        return bool(self.get_dpdk_version() >= "19.11.0")  # appease the type checker
```

Comment:
avoid writing comments in the same line. It's hard to read.

Reply:
fixed